content/en/docs/concepts/containers/images.md

```diff
@@ -157,6 +157,30 @@ that Kubernetes will keep trying to pull the image, with an increasing back-off
 Kubernetes raises the delay between each attempt until it reaches a compiled-in limit,
 which is 300 seconds (5 minutes).
 
+## Serial and parallel image pulls
+
+By default, kubelet pulls images serially. In other words, kubelet will send only
+one image pull request to the image service at a time. Other image pull requests
+have to wait until the one being processed is complete.
+
+If you would like to enable parallel image pulls, you can set the field
+`serializeImagePulls` to false in kubelet configuration. With `serializeImagePulls`
+set to false, image pull requests will be sent to the image service immediately,
+and multiple images will be pulled at the same time.
+
+### Maximum parallel image pulls
+{{< feature-state for_k8s_version="v1.27" state="Alpha" >}}
+By default, when `serializeImagePulls` is set to false, kubelet does not limit the
+maximum number of images being pulled at the same time. If you would like to limit
+the number of parallel image pulls, you can set the field `maxParallelImagePulls`
```

Comment:
This bare documentation it technically OK and correct. So lgtm at will.

What I would suggest as an improvement is to explain why one will want to limit this number

Reply:
Added a short paragraph on why a user might want to limit this number